config/deploy/staging.rb

```diff
@@ -31,7 +31,9 @@
 # http://capistranorb.com/documentation/getting-started/configuration/
 # Feel free to add new variables to customise your setup.
 
-
+set :default_env, {
```

Comment:
Since `RAILS_ENV` is to `production` in both staging and production, I'm using this to override what environment New Relic thinks it is. This will help keep staging and production data separate in New Relic.